experimental/segmenter/src/line.rs

```diff
@@ -153,10 +153,9 @@ pub type LineBreakIteratorUtf16<'l, 's> = LineBreakIterator<'l, 's, LineBreakTyp
 /// listed in rules [LB4][LB4] and [LB5][LB5], special-casing the end of text
 /// according to [LB3][LB3].
 ///
-/// Note that contrary to the grapheme, word, and sentence segmenters, the
-/// breaks returned by this segmenter do not determine a partition of the text
-/// into meaningful segments.  In particular, there is no break opportunity at
-/// the start of text.
+/// For consistency with the grapheme, word, and sentence segmenters, there is
+/// always a breakpoint returned at index 0, but this breakpoint is not a
+/// meaningful line break opportunity.
```

Comment:
FYI: The end of the string is just as special as the start of the string. For line breaking, the end of the string is probably a *paragraph* break, not a *line* break. For an API it still makes sense to return both -- and that's why I don't understand arguments for treating them differently.